helix-core/src/main/java/org/apache/helix/task/api/LocalJarLoader.java

```diff
@@ -0,0 +1,54 @@
+package org.apache.helix.task.api;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LocalJarLoader implements JarLoader {
+  private static final Logger LOG = LoggerFactory.getLogger(LocalJarLoader.class);
+
+  /**
+   * Loads a local JAR file and returns its URL.
+   * @param jarPath path of the JAR file
+   * @return URL of the JAR file at path jarPath
+   */
+  @Override
+  public URL loadJar(String jarPath) {
```

Comment:
Question: What would the returned URL look like for a local JAR?

Reply:
Do you mean to add the example URL to the code? An example is: `file:/home/rabashiz/apache-helix/helix/helix-core/src/test/resources/Reindex.jar`

Reply:
Thank you. That makes sense.